src/openklant/components/klanten/api/serializers.py

```diff
@@ -348,6 +351,8 @@ def create(self, validated_data):
         adres_data = validated_data.pop("adres", None)
         klant = super().create(validated_data)
 
+        # TODO verify Klant with BRP or KvK?
```

Comment:
Ik weet niet of dit alleen bij `create` zou moeten, of misschien op de `save` van het model. Ik weet ook nog niet wat voor check dit precies moet zijn (klant opzoeken in HaalCentraal?)

Reply:
Het verifieren van bijvoorbeeld BSN in HaalCentraal zou een vervolgstap kunnen zijn, maar is nu nog niet in scope. Vooralsnog moet de applicatie zelf aangeven of een klant is geverifieerd of niet (kan ook een medewerker zijn die een klant aanmaakt bijvoorbeeld).

Reply:
Dan zal ik hem voor nu niet immutable maken :p